### PR TITLE
Nginx teardown

### PR DESCRIPTION
**Pull request by notjames**

### CODE REVIEW

File: clusterctl/examples/ssh/bootstrap_scripts/centos-7.4/sds/common_functions.template [hunk 1]
```diff
@@ -390,10 +390,10 @@
               fi
             fi
 
-            all_dockers=($(rpm -qa 'docker*'))
-            yum remove -y kubeadm kubectl kubelet kubernetes-cni "${all_dockers[@]}"
+            yum remove -y kubeadm kubectl kubelet kubernetes-cni nginx* docker*
```
Comment (davidewatson) [inline clusterctl/examples/ssh/bootstrap_scripts/centos-7.4/sds/common_functions.template]: Won't these globs be expanded before yum gets a chance?

Comment (notjames) [reply clusterctl/examples/ssh/bootstrap_scripts/centos-7.4/sds/common_functions.template]: yeah. Lemme fix that.

Comment (notjames) [reply clusterctl/examples/ssh/bootstrap_scripts/centos-7.4/sds/common_functions.template]: Fixed

**Comment by notjames** [comment]
This should still be merged in.

**Pull request closed without merging**